second-readme.md

```diff
@@ -0,0 +1,15 @@
+# Preparation work
```

Comment:
I think _Course preparation_ is a more appropriate header for these items and we should call the file `course-prep.md`. This will need to be reflected in all of the links back to this file.